### PR TITLE
Added SVN prompt

### PR DESCRIPTION
**Pull request by ovidiub13**
If you don't like the output, please propose a different one, and I'll happily adjust.

### CODE REVIEW

**Comment by caiogondim** [comment]
Amazing PR.
Would you mind adding instructions to README?

**Comment by caiogondim** [comment]
@ovidiub13 Could you rebase your branch so i can merge it?

**Comment by ovidiub13** [comment]
I'll give it e try now. Sorry for the delay.

**Comment by ovidiub13** [comment]
Rebase completed.

**Comment by dawikur** [comment]
:+1:

File: bullet-train.zsh-theme [hunk 1]
```diff
@@ -568,11 +603,15 @@ build_prompt() {
...
   prompt_git
+  prompt_svn
   prompt_hg
   prompt_cmd_exec_time
   prompt_end
 }
 
+  prompt_end
+}
+
```
Comment (dawikur) [inline bullet-train.zsh-theme]: This shouldn't be here. I ques you didn't checked this file. This needs to be fixed before merging.

**Comment by dawikur** [comment]
Also I wouldn't be happy with `$(svn_get_rev_nr)`. For me this is in most cases useless. Far better would be using `$(svn_get_branch_name)`. We can use same trick as I have added for git #111. We only need to agree what should be default value.

Still - great stuff.

**Comment by ovidiub13** [comment]
I agree the revision number is pretty useless. At my workplace I don't really use branches, so that's not extremely useful for me. But I can see how it may be useful to others.

I'll check it when I can.

**Comment by dawikur** [comment]
@ovidiub13 Any update on this?

**Comment by ovidiub13** [comment]
I'm trying to find time to handle this... Sorry for the delay...

**Comment by dawikur** [comment]
@ovidiub13  can you update your PR to changes introduced in #188? If not I will be happy to recreate it

**Comment by ovidiub13** [comment]
Apparently I forgot about this PR and deleted my fork.

If there is some desire for it (an issue opened or something) you may ping me and I'll look into it, but otherwise, I see no point in remaking this.

I don't currently use this patch anymore since the rev. no. is not at all useful, and I cannot test the branch, since in the projects where I use SVN, we use it in a non standard way...

**Pull request closed without merging**